Podcast/SearchSectionHeaderView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SearchSectionHeaderTableViewCell.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 4/6/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+protocol SearchTableViewHeaderDelegate: class {
+    func searchTableViewHeaderDidPressViewAllButton(view: SearchSectionHeaderView)
+}
+
+class SearchSectionHeaderView: UIView {
```

Comment:
![image](https://user-images.githubusercontent.com/20246620/38657019-0b6ffeb8-3dec-11e8-80ea-857a3b58d67f.png)
i'm getting a weird space (and some constraint issues) when dismissing the header

Reply:
See what I did in my current branch to remove this gap .. you have to reset constraints and make the header view nil